### ISSUE TITLE
Fix labels / colors after #174, #177

### ISSUE THREAD

**Issue by mkoeppe** [sagemath/trac-to-github#178]
https://github.com/sagemath/sage/labels?page=3&sort=name-asc
<img width="121" alt="Screenshot 2023-02-04 at 10 07 41 PM" src="https://user-images.githubusercontent.com/8345221/216804579-4fbd76ea-e8c7-4f71-ba78-80844911089d.png">

**Comment by mkoeppe** [comment]
@kwankyu

**Comment by kwankyu** [comment]
I get 404 from the link. What's the problem? The labels don't get the texts we intended? It is "wontfix" instead of "r: wontfix"?

**Comment by mkoeppe** [comment]
The screenshot shows both prefixed and non-prefixed labels, and the `r: ` labels have the wrong color.

**Comment by mkoeppe** [comment]
I think I'll just back out the most recent changes to the labels.

**Comment by kwankyu** [comment]
Ah, yes. I don't quite understand how labels are defined. Let me ask some questions.

(1) Are non-prefixed labels remnants from the previous import?

(2) Are they completely defined only by `migrate.cfg.sagetracmigrationarchive`?

**Comment by mkoeppe** [comment]
(1) No, I always clean in the production environment

(2) No, there are also some default colors defined somewhere in the script

**Comment by kwankyu** [comment]
Then "duplicate and "r: duplicate" are both defined somewhere in our migration script?

**Comment by mkoeppe** [comment]
One mistake is in `map_resolution`, I think - there's no prefix added

**Comment by mkoeppe** [comment]
Maybe this would already fix the colors too, not sure.

**Comment by mkoeppe** [comment]
OK, I've made this change in 6c41c5d

**Comment by kwankyu** [comment]
If we can use GHE, which seems down now, we can experiment.

**Comment by kwankyu** [comment]
> Maybe this would already fix the colors too, not sure.

This might be enough.

**Comment by mkoeppe** [comment]
Yes, our trial license for GHE has already expired, Dima shut down the server a few days ago.

**Comment by mkoeppe** [comment]
Looks like this is indeed fixing it already.

**Comment by kwankyu** [comment]
If `label_category` is `r: resolution`, then this code
```
labelcolor[label_category] 
```
will not work since we only have `resolution: c6c6c6` but not `r: resolution: c6c6c6` in `migrate.cfg.sagetracmigrationarchive`. Is `label_category` still `resolution`?

**Comment by kwankyu** [comment]
Now I saw it. OK!

**Comment by kwankyu** [comment]
Do these work as intended?
```
milestones_to_labels = {
 'sage-duplicate/invalid/wontfix': 'r: invalid',
 'sage-duplicate/invalid': 'r: invalid',
 'sage-duplicate': 'r: duplicate',
```

I don't see  where these labels get the correct colors.

**Comment by kwankyu** [comment]
If not, we need this:
```diff
diff --git a/migrate.cfg.sagetracmigrationarchive b/migrate.cfg.sagetracmigrationarchive
index 3d0ba93e07..36e60b014c 100644
--- a/migrate.cfg.sagetracmigrationarchive
+++ b/migrate.cfg.sagetracmigrationarchive
@@ -216,6 +216,8 @@ label_colors = {
  't: refactoring': '696969',
  't: feature': '696969',
  't: tests': '696969',
+ 'r: invalid': 'c6c6c6',
+ 'r: duplicate': 'c6c6c6',
  'resolution': 'c6c6c6'}
 
 # values to be ignored on issue conversion
```

**Comment by mkoeppe** [comment]
I have checked the output (`archive/repositories_000001.json`). They are correct.

**Comment by mkoeppe** [comment]
I think it happens to go through the map_resolution path first, and in this way the label picks up the correct color.

**Comment by kwankyu** [comment]
Okay then.
